installation/README.md

```diff
@@ -0,0 +1,93 @@
+# Installing Config Sync manually
+
+This document is a guide on how to manually install Config Sync on a Kubernetes
+cluster without Fleet Management.
```

Comment:
```suggestion
cluster.
```

Reply:
Updated in https://github.com/GoogleContainerTools/kpt-config-sync/pull/1456